core/misc/tabledrag.js

```diff
@@ -123,6 +133,15 @@ Backdrop.tableDrag = function (table, tableSettings) {
   $(document).on('touchend', function (event) { return self.dropRow(event.originalEvent.touches[0], self); });
   $(document).on('mousemove pointermove', function (event) { return self.dragRow(event, self); });
   $(document).on('mouseup pointerup', function (event) { return self.dropRow(event, self); });
+  // React to localStorage event showing or hiding weight columns.
+  $(window).bind('storage', $.proxy(function (e) {
```

Comment:
Function jQuery bind is also [deprecated](https://api.jquery.com/bind/).

Reply:
Fixed with [`c5723f7` (#3870)](https://github.com/backdrop/backdrop/pull/3870/commits/c5723f7f78571004fac900e7c5eb4ed63ee5a248)